### ISSUE TITLE
`no-empty-file` support for .html

### ISSUE THREAD

**Issue by daaa57150** [sindresorhus/eslint-plugin-unicorn#2042]
I tried to start using this plugin as it has some interesting rules, I was especially interested in the `no-empty-file` rule for our `scss` files.
But when I added the rule in the section verifying the scss, this error happened:
```
An unhandled exception occurred: Cannot read properties of undefined (reading 'some')
Occurred while linting C:\some\path\src\app\components\commons\continue-button.component.ts\1_inline-template-continue-button.component.ts-1.component.html:1
Rule: "unicorn/no-empty-file"
``` 
As it happens, I'm applying a list of rules on many types of files, including `html`:
```json
    {
      "files": [
        "*.ts", "*.html", "*.scss"
      ],
      "rules": {
        "other rules here": "error",
        "unicorn/no-empty-file": "error"
      }
    }
```
And it seems that the `no-empty-file` rule just dislikes the `html` files. I removed the `html` from the list: no error, I replaced the rule with `no-instanceof-array`: no error.

**Minimal config to make it fail:**
```json
    {
      "files": ["*.html"],
      "rules": {
        "unicorn/no-empty-file": "error"
      }
    }
```

It looks like this rule is not meant to check any types of files ? Because even if I remove the `html` in the list, my empty `scss` are not flagged by the rule.

**Comment by fregante** [comment]
Not supported at the moment, see:

- https://github.com/sindresorhus/eslint-plugin-unicorn/issues/2181
